code/ship/ship.cpp

```diff
@@ -2665,6 +2666,9 @@ int parse_ship_values(ship_info* sip, bool first_time, bool replace)
 	//Parse the engine sound
 	parse_sound("$EngineSnd:", &sip->engine_snd, sip->name);
 
+	if(optional_string("$Minimum Engine Volume:"))
+		stuff_float(&sip->min_engine_vol);
```

Comment:
Ahh, the formula would need to be documented, then users can figure out how different values would change the behavior.